core/src/main/scala/cromwell/core/logging/LoggingTest.scala

```diff
@@ -0,0 +1,7 @@
+package cromwell.core.logging
+
+import java.util.regex.Pattern
+
+object LoggingTest {
+  def escapePattern(pattern: String) = Pattern.quote(pattern)
```

Comment:
Thanks for the detailed explanation ! Looks like backends have the dependency on core test transitively through the "backend" project. So it actually works now.